eras/shelley/impl/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Version history for `cardano-ledger-shelley`
 
-## 1.12.2.0
+## 1.13.0.0
```

Comment:
What is the breaking change that warrants a bump to `1.13.0.0` instead of `1.12.3.0`?

Reply:
I know that PVP dictates that orphan instances warrant a major version bump, but in this case any instances that we put into testlibs do not follow this rule. So `Arbitrary` and `ToExpr` should be treated as regular, not orphan instances. They are defined for types in this package and they are only defined as orphans because they must live in the `testlib`

Reply:
Ah, okay. I'll only bump the minor version then, thanks!